src/main/kotlin/no/nav/familie/ks/sak/kjerne/tilbakekreving/TilbakekrevingService.kt

```diff
@@ -98,6 +100,33 @@ class TilbakekrevingService(
     fun sendOpprettTilbakekrevingRequest(behandling: Behandling) =
         tilbakekrevingKlient.opprettTilbakekrevingBehandling(lagOpprettTilbakekrevingRequest(behandling))
 
+    fun opprettTilbakekrevingsbehandlingManuell(fagsakId: Long) {
```

Comment:
flisespikking men `opprettTilbakekrevingsbehandlingManuell ` -> `opprettTilbakekrevingsbehandlingManuelt ` for konsistens av navngiving